_data/companies.yml

```diff
@@ -0,0 +1,803 @@
+---
+- name: AISA Digital
+  id: aisa-digital
+  type: [ mt-provider ]
+  location: Singapore
+  foundation: 2018
+  urls:
+    - https://www.aisa.digital/
+    - https://www.crunchbase.com/organization/aisa-digital
+
+- name: Alexa Translations A.I.
+  id: alexa-translations-ai
+  type: [ mt-provider ]
+  location: Canada
+  foundation: 2002
+  urls:
+    - https://alexatranslations.com/
+    - https://www.crunchbase.com/organization/alexa-translations-a-i
+
+- name: Alibaba Cloud Computing
+  id: alibaba-cloud
+  type: [ technology-provider ]
+  location: China
+  foundation: 2009
+  urls:
+    - https://www.alibabacloud.com/
+    - https://www.crunchbase.com/organization/alibaba-cloud
+
+- name: Amazon
+  id: amazon
+  type: [ technology-provider ]
+  location: United States
+  foundation: 1994
+  urls:
+    - https://aws.amazon.com/translate/
+
+- name: Amplexor
+  id: amplexor
+  type: [ lsp ]
+  location: Luxembourg
+  foundation: 2001
+  urls:
+    - https://www.acolad.com/
```

Comment:
I think better not to put this, just let them click through to the Acolad entry.

Reply:
sure